01-front-end-basics/jquery-dom-manipulation/24/README.md

```diff
@@ -0,0 +1,252 @@
+
+#### Code Block #1:
+
+<img src="https://cloud.githubusercontent.com/assets/7833470/10717913/b94098a2-7b22-11e5-8115-decf62e0b65b.png">
+
+#### Code Block #2:
+
+<img src="https://cloud.githubusercontent.com/assets/7833470/10717912/b9405ed2-7b22-11e5-9a64-07a19d7473d7.png">
+
+#### Code Block #3:
+
+<img src="https://cloud.githubusercontent.com/assets/7833470/10717909/b93ebabe-7b22-11e5-95a6-605fb5ca9312.png">
+
+#### Code Block #4:
+
+<img src="https://cloud.githubusercontent.com/assets/7833470/10717911/b940228c-7b22-11e5-84f5-0ee1b941f45e.png">
+
+#### Code Block #5:
+
+<img src="https://cloud.githubusercontent.com/assets/7833470/10717910/b93fc8dc-7b22-11e5-8fc7-44d9f96e3fab.png">
+
+### jQuery Selectors
+
+1. Use jQuery to select the element containing the words "Hello There" with as many different selectors as you can think of:
+
+  ``` html
+  <!DOCTYPE html>
+  <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Example 1</title>
+  </head>
+  <body>
+    <div id="greeting">Hello There</div>
+  </body>
+  </html>
+  ```
+
+2. How would you *get* the value "Hello There"?
+3. How would you change (or *set*) the value to "Hola"?
+4. How would you add an `<h1>` to the page that says "A Spanish Greeting"?
+5. How would you add a second greeting ("Que tal?") below the first?
+6. How would you add the class "section" to both greetings?
+
+### Indiana Jones And the Temple of DOM
+
+Go to the <a href="https://generalassemb.ly" target="_blank">General Assembly homepage</a>, and open your Chrome Developer Console.
+
+1. Select the huge text splashed across the front page.
+  * *Get* its text.
+  * Change (*set*) its text to "Indiana Jones and the Temple of DOM"
+  * Add a `<small>` tag containing the text "(Rated G)" to the end of the line.
+2. Turn the big red button blue.
+3. Select the image to the right of the "Full-Time Courses" section (the one with the red-cushioned chair).
+  * *Get* its image `src` (image url).
+  * Swap out the image for this <a href="https://cloud.githubusercontent.com/assets/7833470/10717957/17c7ef00-7b24-11e5-8fea-4484b908f8a7.gif" target="_blank">GIF of Indiana Jones</a>.
+4. Change the text of *every* paragraph on the page to: "Snakes. Why did it have to be snakes?"
+5. **Bonus:** Combine all your code into a single function called `changeAllTheThings`. When you call the function, it should make all your changes to the page. (You'll want to use a Script Snippet in your Chrome Developer Console for this. `cmd + option + J > Sources Tab > Snippets Sub-Tab`)
+
+## Stretch Challenge
+
+Open up this <a href="http://jsfiddle.net/cameronjacoby/Lfnkg20h/1/" target="_blank">JSFiddle</a>. Write jQuery that makes the green boxes fade out one by one. The boxes should disappear in order from left to right, and there should be a one second delay before the next one goes away.
```

Comment:
IMO `$(document).on('ready',` is more consistent with other event binding, less "magic".

For example, good luck googling "$(function() {})"!